src/tests/support.cpp

```diff
@@ -672,3 +672,60 @@ check_json_field_bool(json_object *obj, const std::string &field, bool value)
     }
     return json_object_get_boolean(fld) == value;
 }
+
+pgp_key_t*
+rnp_tests_get_key_by_id(const rnp_key_store_t* keyring, const std::string& keyid, pgp_key_t* after)
+{
+    pgp_key_t *key = NULL;
+    uint8_t keyid_bin[PGP_KEY_ID_SIZE];
```

Comment:
Since we are migrating to C++ then could you please use C++ vector here (and in other places where new code is added)?